### ISSUE TITLE
When editing an item most everything disappears 

### ISSUE THREAD

**Issue by PerpSearch** [koillection/koillection#1171]
I haven't done anything to it I don't think, but when I went to edit a record, all the data fields are gone.  I can go back and still see them but I can't edit what's not populated.    This might help. https://imgur.com/xydhxSf.mp4 

I tried to restore a backup from yesterday and it didn't fix the issue so I tried stopping and restarting the container. That was a mistake. ~~Now it's in a loop starting and immediately exiting.~~  I've attached logs for the two containers.  

I did get it back up and running w/ a backup from a couple days ago, but the weird behavior from the above video is still a problem.  The last pair of logs are the most current.  

(edit)    Another data point:  I tried adding a new item and all the data fields are missing. Additional fields selection, Load Common Fields and Load Collection Fields do nothing.   I can add a picture and tags but that's it.  Hope that helps a little.

[_Koillection_logs.txt](https://github.com/user-attachments/files/16103192/_Koillection_logs.txt)
[_Koillection-DB_logs.txt](https://github.com/user-attachments/files/16103193/_Koillection-DB_logs.txt)

Now that it's running again
[_Koillection-DB_logs.txt](https://github.com/user-attachments/files/16103256/_Koillection-DB_logs.txt)
[_Koillection_logs.txt](https://github.com/user-attachments/files/16103257/_Koillection_logs.txt)

**Comment by benjaminjonard** [comment]
It looks like a JavaScript error, nothing should be lost as long as you don't save anything. 
Is there any error in your browser console ? (Right click on the page -> Inspect)

**Comment by PerpSearch** [comment]
You might be on to something there. I just tried it on my mobile and I was able to access normally. I'll check the browser tomorrow as it's almost 2 am here. I typically run it in Firefox, but run chrome along also and it never occurred to me to try a different browser through my android phone may be proof enough. Back tomorrow.
Thanks.

**Comment by PerpSearch** [comment]
Well that's embarrassing. After my last reply I started wondering what could possibly cause all that stuff to just become inaccessible and without data loss.  So it gnawed at me til I had a moment of clarity about 3AM.  

AdBlockPlus decided that the edit function behaved too much like an invasive popup advertisement and blocked it.  

I made no changes so I'm guessing the last automatic update changed it for me.  
Problem solved and I guess it's something to consider when someone else turns up with equally weird symptoms. 

Thanks again and sorry for the false alarm.
C
